packages/studio-plugin/src/utils/ComponentTreeHelpers.ts

```diff
@@ -186,4 +186,44 @@ export default class ComponentTreeHelpers {
 
     return descendants;
   }
+
+  /**
+   * Returns the component UUID that is closest common parent of the two target components.
+   * This is similar to the lowest common ancestor except it cannot be one of the targets.
+   */
+  static getLowestParentUUID(
+    targetOneUUID: string | undefined,
+    targetTwoUUID: string | undefined,
```

Comment:
when would either of these UUIDs be undefined?

Reply:
when it is the root (which i guess would never happen normally but when I start passing in the parentUUIDs then it will be needed)

Reply:
actually I just realized that without the undefined, putting `selectedComponentUUIDs.at(0)` as an argument gives an error, so I added it back